mobile/android/app/src/main/java/com/companionkit/MainApplication.java

```diff
@@ -7,6 +7,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.reactnative.googlefit.GoogleFitPackage;
```

Comment:
Would it be better (and is it possible) to import this only for Android?

Reply:
This is actually an Android file (MainApllication.java). In terms of the .tsx files I'll have to look into it